sdk/batch/Azure.ResourceManager.Batch/CHANGELOG.md

```diff
@@ -1,15 +1,25 @@
 # Release History
 
-## 1.1.0-beta.1 (Unreleased)
+## 2.0.0 (2022-11-07)
 
-### Features Added
+### Features
+
+- Added new custom enum type `NodeCommunicationMode`.
+  - This property determines how a pool communicates with the Batch service.
+  - Possible values: Default, Classic, Simplified.
+- Added properties `CurrentNodeCommunicationMode` and `TargetNodeCommunicationMode` of type `NodeCommunicationMode` to `BatchAccountPoolData`.
 
 ### Breaking Changes
 
-### Bugs Fixed
+- Modified casing of Username property of the `BatchCifsMountConfiguration` class to `UserName`.
+- Modified casing of DynamicVNetAssignmentScope property of the `BatchNetworkConfiguration` class to `DynamicVnetAssignmentScope`.
+- Modified casing of ActionRequired property of the `BatchPrivateLinkServiceConnectionState` class to `ActionsRequired`.
```

Comment:
Breaking changes are not allowed. Please use configuration or customized code to make them backward compatible.

Reply:
@fengzhou-msft Reverted back breaking changes.